### DISCUSSION TITLE
I can't start Asreview anymore

### DISCUSSION THREAD

**Discussion by Koopmanon** [asreview/asreview#1532] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hello everyone. I use Asreview Lab to assist me in my systematic literature review. I had a small break from this project and did not open asreview lab for about 3 months. Now I cannot open it anymore. I already installed it again, but keep getting the same reaction from cmd.exe

C:\Users\82278mko>asreviewlab
'asreviewlab' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\82278mko>asreview lab
'asreview' is not recognized as an internal or external command,
operable program or batch file.

What do I do? And bear in my mind that I am not very good with computers at all. Thanks in advance.

**Comment by Rensvandeschoot** [comment, marked as answer]
Can you try the solution from the [troubleshooting](https://github.com/asreview/asreview/assets/36502709/a8348dd8-2282-49a4-90f1-157fc10150de) to start ASReview: 

``` 
python -m pip install asreview
python -m asreview lab
```

**Reply by Koopmanon** [reply]
That worked, thank you so much!
